packages/cli/lib/services/deploy.service.ts

```diff
@@ -365,7 +358,7 @@ class DeployService {
             return null;
         }
 
-        return { flowConfigs: postData, postConnectionScriptsByProvider, jsonSchema: jsonSchema };
+        return { flowConfigs: postData, postConnectionScriptsByProvider, jsonSchema: JSON.parse(jsonSchema) as JSONSchema7 };
```

Comment:
parsing can fail. are we ok throwing here?

Reply:
good point